### DISCUSSION TITLE
Bibliography does not show up

### DISCUSSION THREAD

**Discussion by monica-dyreby** [joaomlourenco/novathesis#449] [Q&A]
Hello,
When I compile the contents of the zipped file the bibliography doesn't show up. 

I have not changed the file at all. 
I am using MiKTex and TexStudio. I have also changed the bibliography tool to Biber.
When I look at the log file I see warnings like these:
No file template.bbl.
citation 'flex'(biblatex) was not found at all in the .bbl file,(biblatex) bibliography labels cannot be generated,(biblatex) please correct the citation.

Thanks in advance

**Comment by joaomlourenco** [comment]
If the bibliography does not show up it is because TexStudio id not properly configured or you are not processing the file properly.

I suggest you configure TexStudio to use `latexmk` and not pdflatex.

**Comment by monica-dyreby** [comment]
I am using latexmk: 
![image](https://github.com/joaomlourenco/novathesis/assets/139464597/57a2eef2-f3b7-4ec4-9c60-562afb87b1f0)
I downloaded the zipped folder from: https://joaomlourenco.github.io/novathesis/
Unzipped it and opened the template file in TexStudio and compiled it.
Is there a step I missed?

**Reply by vicenters1** [reply]
I had the same problem and simply restarted the computer and it fixed itself.

ty jerry
